### PR TITLE
PXD-1501 Compose-Services Sheepdog Fix

### PR DESCRIPTION
**Pull request by stevencui729**
Updated sheepdog and fence to allow for data submission
Updated README with instructions for user syncing and updating images
Updated docker-compose.yaml to use master tag images for everything but sheepdog and peregrine

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,14 +34,21 @@ Setup the credentials with the provided script by running:
...
 
 ### Setting up Google API Credentials for Fence
-To set up Google API Credentials, go to the [Google Developer Console](https://console.developers.google.com/apis/credentials) and click the 'Create Credentials' button. Follow the prompts to create a new OAuth Client ID for a Web Application, and then copy your client ID and client secret and use them to fill in the 'google_client_secret' and 'google_client_id' fields in the fence section of the `api_configs/creds.json` JSON file. 
+To set up Google API Credentials, go to the [Google Developer Console](https://console.developers.google.com/apis/credentials) and click the 'Create Credentials' button. Follow the prompts to create a new OAuth Client ID for a Web Application, and then copy your client ID and client secret and use them to fill in the 'google_client_secret' and 'google_client_id' fields in the `api_configs/fence_creds.json` JSON file.
```
Comment (zflamig) [inline README.md]: Please add a note about enabling the Google+ API too

File: apis_configs/user.yaml [hunk 2]
```diff
@@ -10,64 +10,4 @@ users:
     projects:
     - auth_id: acct
```
Comment (zflamig) [inline apis_configs/user.yaml]: acct -> project1 to be more clear what it represents?

File: apis_configs/fence_creds.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+  "db_host": "postgres",
+  "db_username": "fence_user",
+  "db_password": "fence_pass",
+  "db_database": "fence_db",
+  "hostname": "localhost",
```
Comment (zflamig) [inline apis_configs/fence_creds.json]: Lets add a note about updating this too if you are running it on a remote machine (i.e. EC2 instance)

Comment (stevencui729) [reply apis_configs/fence_creds.json]: Is this referring to trying to run the Docker Compose setup in production on a remote host? I haven't tried this, I think there might be more things to tweak, such as adding a Swarm setup.

Comment (stevencui729) [reply apis_configs/fence_creds.json]: So, this would essentially just be changing the hostname?

Comment (zflamig) [reply apis_configs/fence_creds.json]: Yep, we just need a note in the readme on how to set this up and where to change it

**Pull request merged**